tests/containers/rootless_podman.pm

```diff
@@ -55,7 +55,7 @@ sub run {
     }
 
     # Prepare for Podman 3.4.4 and CGroups v2
-    if (is_sle('15-SP3+') || is_leap('15.3+')) {
+    if ((is_sle('15-SP3+') || is_leap('15.3+')) && !check_var('CONTAINERS_CGROUP_VERSION', '1')) {
```

Comment:
```suggestion
    if ((is_sle('15-SP3+') || is_leap('15.3+')) && check_var('CONTAINERS_CGROUP_VERSION', '2')) {
```
If I get the condition right, this should be exclusively for cgroup version 2, and not just for "not version 1". I'm especially looking at the value 0 here, which otherwise might cause issues, if not set.

Reply:
I use `!` in case the variable is not set.

Reply:
Functionally ok, but I find it confusing. Feel free to leave it as it is though.

Reply:
It's not just style. Consider the case where `CONTAINERS_CGROUP_VERSION` is not set. :)